internal/payloads/instance_description_payload.go

```diff
@@ -0,0 +1,28 @@
+package payloads
+
+import (
+	"net/http"
+
+	"github.com/RHEnVision/provisioning-backend/internal/clients"
+	"github.com/go-chi/render"
+)
+
+type InstanceDescriptionResponse struct {
+	*clients.InstanceDescription
+}
+
+func (s *InstanceDescriptionResponse) Bind(_ *http.Request) error {
+	return nil
+}
```

Comment:
If there is no payload for request, you do not need any `Bind` method. You can just delete this.